api-channel-jira/src/main/java/com/synopsys/integration/alert/api/channel/jira/distribution/custom/MessageReplacementValues.java

```diff
@@ -25,17 +21,24 @@
     private final String componentVersionName;
     private final String severity;
 
-    public static JiraCustomFieldReplacementValues trivial(LinkableItem provider) {
-        return new JiraCustomFieldReplacementValues(provider.getLabel(), DEFAULT_REPLACEMENT, null, null, null, null);
+    public static MessageReplacementValues trivial(String providerLabel, String projectName) {
+        return new MessageReplacementValues(
+            providerLabel,
+            projectName,
+            MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE,
+            MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE,
+            MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE,
+            MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE
```

Comment:
Is there a difference between these being `DEFAULT_REPLACEMENT` and `null`? I have a feeling behavior will change.

Reply:
If you check master the original functionality would use null initially in this method then change it to none a couple steps later. So we were always pushing none up to Jira
```
    private static String replaceFieldValues(String originalFieldValue, JiraCustomFieldReplacementValues replacementValues) {
        String replacedFieldValue = StringUtils.replace(originalFieldValue, REPLACEMENT_PROVIDER_NAME, replacementValues.getProviderName());
        replacedFieldValue = StringUtils.replace(replacedFieldValue, REPLACEMENT_PROJECT_NAME, replacementValues.getProjectName());
        replacedFieldValue = StringUtils.replace(replacedFieldValue, REPLACEMENT_PROJECT_VERSION, replacementValues.getProjectVersionName().orElse(JiraCustomFieldReplacementValues.DEFAULT_REPLACEMENT));
        replacedFieldValue = StringUtils.replace(replacedFieldValue, REPLACEMENT_COMPONENT_NAME, replacementValues.getComponentName().orElse(JiraCustomFieldReplacementValues.DEFAULT_REPLACEMENT));
        replacedFieldValue = StringUtils.replace(replacedFieldValue, REPLACEMENT_COMPONENT_VERSION, replacementValues.getComponentVersionName().orElse(JiraCustomFieldReplacementValues.DEFAULT_REPLACEMENT));
        replacedFieldValue = StringUtils.replace(replacedFieldValue, REPLACEMENT_SEVERITY, replacementValues.getSeverity().orElse(JiraCustomFieldReplacementValues.DEFAULT_REPLACEMENT));
        return replacedFieldValue;
    }
```
Since this was originally intended for fields, perhaps there's a better alternative? Do we want to insert 'None' into the variables when none is found or empty? Empty might be confusing imo

Reply:
Ok, this probably makes sense because if a field is required, it might not be able to be created without a value.